### ISSUE TITLE
Upgrade Tile downloading Small Footprint Runtime

### ISSUE THREAD

**Issue by gaigaslab** [pivotal-cf/pcf-pipelines#210]
When using the upgrade-tile pipeline to upgrade elastic-runtime, it downloads and performs SHA256 on both cf and srt even though cf is the only product being specified in the YAML:

```
2017/10/26 13:21:01 PivNet Resource version: 0.31.4
2017/10/26 13:21:01 Creating download directory: /tmp/build/get
2017/10/26 13:21:01 Getting release for product slug: 'elastic-runtime' and product version: '1.12.5'
2017/10/26 13:21:02 Accepting EULA for release with ID: 7440
2017/10/26 13:21:03 Getting product files
2017/10/26 13:21:03 Getting file groups
2017/10/26 13:21:05 Getting release dependencies
2017/10/26 13:21:05 Getting dependency specifiers
2017/10/26 13:21:05 Getting release upgrade paths
2017/10/26 13:21:06 Getting upgrade path specifiers
2017/10/26 13:21:06 Downloading files
2017/10/26 13:21:06 Filtering download links by glob
2017/10/26 13:21:06 Downloading filtered files
2017/10/26 13:21:06 Downloading: 'PCF Small Footprint Runtime' to file: '/tmp/build/get/srt-1.12.5-build.1.pivotal'
 6.82 GiB / 6.82 GiB [===========================================] 100.00% 4m23s
2017/10/26 13:25:30 Downloading: 'PCF Elastic Runtime' to file: '/tmp/build/get/cf-1.12.5-build.1.pivotal'
 6.82 GiB / 6.82 GiB [===========================================] 100.00% 5m47s
2017/10/26 13:31:19 Calculating SHA256 or MD5 for downloaded files
2017/10/26 13:32:51 /tmp/build/get/srt-1.12.5-build.1.pivotal SHA256 is: cb7db8afd44edd6ac48f862ac9e60f7d4db6b33e322b6d89e9aea9c8f58bfd0b
2017/10/26 13:34:17 /tmp/build/get/cf-1.12.5-build.1.pivotal SHA256 is: d5777a5686b1cd0a8e2de36b8092f11ccf09ec3bbd356365d009f4dca5460432
2017/10/26 13:34:17 SHA256 or MD5 matched for all downloaded files
2017/10/26 13:34:17 Get complete
2017/10/26 13:34:17 Creating metadata
2017/10/26 13:34:17 Writing metadata files
```

YAML fragment:
```
product_slug: elastic-runtime
product_name: cf
```

**Comment by krishicks** [comment]
This is because it downloads all `*.pivotal` files. The glob regex should be changed to be `cf-*.pivotal` or similar to target only ERT. This would need to be done across all pipelines that try to download ERT.

**Comment by abbyachau** [comment]
Hi @gaigaslab, we've implemented this fix in pcf-pipelines release v21.1. Please review the release notes for how to update this new parameter. https://github.com/pivotal-cf/pcf-pipelines/releases
If any questions/feedback, please create a new issue. Thanks!
